src/Apps/Order/Mutations/useShippingOperations.ts

```diff
@@ -0,0 +1,121 @@
+import { createUserAddress } from "Apps/Order/Mutations/CreateUserAddress"
+import { deleteUserAddress } from "Apps/Order/Mutations/DeleteUserAddress"
+import { setShipping } from "Apps/Order/Mutations/SetShipping"
+import { updateUserAddress } from "Apps/Order/Mutations/UpdateUserAddress"
+import { ShippingProps } from "Apps/Order/Routes/Shipping"
+import { ShippingAddressFormValues } from "Apps/Order/Utils/shippingUtils"
+
+import { useSystemContext } from "System/useSystemContext"
+import createLogger from "Utils/logger"
+import { CreateUserAddressMutation$data } from "__generated__/CreateUserAddressMutation.graphql"
+import {
+  CommerceSetShippingInput,
+  SetShippingMutation$data,
+} from "__generated__/SetShippingMutation.graphql"
+import {
+  UpdateUserAddressMutation$data,
+  UserAddressAttributes,
+} from "__generated__/UpdateUserAddressMutation.graphql"
+import { useCallback } from "react"
+
+export type SaveFulfillmentDetailsResponse = NonNullable<
+  SetShippingMutation$data["commerceSetShipping"]
+>["orderOrError"]
+
+export const useShippingOperations = (
```

Comment:
This hook may not be necessary. I also noticed that the settings page has its own mutations for updating user addresses, and wonder if these should eventually be joined.